templates/components/course-list/course-list.ui_patterns.yml

```
@@ -0,0 +1,45 @@
+course-list:
```

This needs to be underscores, and you'll have to change it in display modes or views that use it.
```suggestion
course_list:
```

---

fixed